backend/src/main/java/ch/nova_omnia/lernello/service/CustomUserDetailsService.java

```diff
@@ -45,6 +45,7 @@ private List<GrantedAuthority> getUserScopes(User user) {
             case INSTRUCTOR -> {
                 scopes.add(new SimpleGrantedAuthority("SCOPE_folders:read"));
                 scopes.add(new SimpleGrantedAuthority("SCOPE_folders:write"));
+                scopes.add(new SimpleGrantedAuthority("SCOPE_media_files:read"));
```

Comment:
instructors should have both read and write permission. trainees also require the read permission (might have to rethink the reading files in the future due to how the browser works btw, but that can wait till then)